executor/load_data.go

```diff
@@ -642,10 +642,13 @@ func (e *LoadDataWorker) commitWork(ctx context.Context) (err error) {
 		tasks               uint64
 		lastScannedFileSize int64
 		currScannedFileSize int64
+		backgroundCtx       = context.Background()
```

Comment:
err msg at line `return errors.Errorf("failed to keepalive for LOAD DATA job %d", jobID)` need updated too, job might be cancelled or deleted

Reply:
https://github.com/pingcap/tidb/pull/42137/commits/24bd2b966531ddef6b050f284c4b3bf2a873f3fa
updated